tabular/src/autogluon/tabular/configs/hyperparameter_configs.py

```diff
@@ -81,7 +81,7 @@
         'CAT': {},
         'XGB': {},
         # 'FASTAI': {},  # FastAI gets killed if the dataset is large (400K rows).
-        'AG_TEXT_NN': ['medium_quality_faster_train'],  # TODO, Support changing the config w.r.t the preset option.
+        'AG_TEXT_NN': {},  # TODO, Support changing the config w.r.t the preset option.
```

Comment:
We can consider to support the presets.

Reply:
If we use name "AG_MM_NN" to indicate Text Predictor, there will be conflicts in naming the Pytorch-based Image Predictor and Image+Text Predictor. I think using name "AG_TEXT_NN" may be a better choice.

Reply:
The previous presets have three choices: low quality, medium quality, and best quality, corresponding to three backbones: electra small, base, and large. These seem not that necessary since we already provide the backbone customization functionality. Users can try any huggingface text backbones. They shouldn't just rely on the naive preset rule.

Reply:
After a second thought, I agree that keeping AG_TEXT_NN is good. Regarding presets, we should add some documentations in the customization tutorial to recommend some configuration choices to the user (e.g. configurations for fast inference and training). It is non-trivial for beginners to search for the huggingface backbone names and set those keys.

Reply:
Agree. We can add some recommendations in the tutorial.

Reply:
I think presets does not conflict with customization. It's just some preselected customization and there shouldn't be a reason we don't support it. It is critical for beginners

Reply:
I think presets does not conflict with customization. It's just some preselected customization and there shouldn't be a reason we don't support it. It is critical for beginners

Reply:
Code updated. Preset is supported now.